### PR TITLE
Swap COS8 image for UBI8 image

### PR DESCRIPTION
**Pull request by ferricoxide**
Attempt to fix CI failures caused by May 31st's  discontinuance of CentOS Stream 8

### CODE REVIEW

File: .github/workflows/test-salt-linux.yml [hunk 1]
```diff
@@ -39,7 +39,8 @@ jobs:
       image: >-
         ${{ fromJson('{
           7: "quay.io/centos/centos:centos7",
-          8: "quay.io/centos/centos:stream8"
+          8: "hub.docker.com/redhat:ubi8",
+          9: "quay.io/centos/centos:stream9"
```
Comment (lorengordon) [inline .github/workflows/test-salt-linux.yml:43]: I think we can just use redhat:ubi9 here also...?

**Review by lorengordon — COMMENTED** [review]
Let's update .bumpversion.cfg with a minor version update, and add a changelog entry.

File: .github/workflows/test-salt-linux.yml (hunk 1 above)
Comment (ferricoxide) [reply .github/workflows/test-salt-linux.yml]: switched

**Comment by ferricoxide** [comment]
> Let's update .bumpversion.cfg with a minor version update, and add a changelog entry.

I updated to 1.6.0 (from 1.5.0)

**Comment by lorengordon** [comment]
> > Let's update .bumpversion.cfg with a minor version update, and add a changelog entry.
> 
> I updated to 1.6.0 (from 1.5.0)

Don't forget the changelog entry!

File: .github/workflows/test-salt-linux.yml [hunk 2]
```diff
@@ -39,7 +39,8 @@ jobs:
       image: >-
         ${{ fromJson('{
           7: "quay.io/centos/centos:centos7",
-          8: "quay.io/centos/centos:stream8"
+          8: "hub.docker.com/redhat:ubi8",
+          9: "hub.docker.com/redhat:ubi9"
```
Comment (lorengordon) [inline .github/workflows/test-salt-linux.yml:43]: based on the error from the ash-linux test, i think the registry needs to be `docker.io`...

```suggestion
          8: "docker.io/redhat:ubi8",
          9: "docker.io/redhat:ubi9"
```

Comment (ferricoxide) [reply .github/workflows/test-salt-linux.yml]: out-of-date: UBIs are a dead-end (they can be _made_ to work, but it's significantly-higher effort – both to get working and, likely, to _keep_ working – than using either Rocky or Alma)

**Review by lorengordon — COMMENTED** [review]
Still need the changelog entry, then good to go.

**Review by lorengordon — APPROVED** [review]

**Pull request merged**